single-node/setup/resources/deployment.toml

```diff
@@ -9,11 +9,11 @@ password = "admin"
 create_admin_account = true
 
 [user_store]
-type = "database"
-connection_url = "ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}"
-connection_name = "uid=admin,ou=system"
-connection_password = "admin"
-base_dn = "dc=wso2,dc=org"
+type = "database_unique_id"
+#connection_url = "ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}"
+##connection_name = "uid=admin,ou=system"
+#connection_password = "admin"
+#base_dn = "dc=wso2,dc=org"
```

Comment:
Why do we need to keep these?

Reply:
These will be help if we need to run the test in other user stores. For JDBC theses configurations not valid.

Reply:
then let's add comments to easily understand that.

Reply:
updated. https://github.com/wso2/performance-is/pull/69